Program.cs

```diff
@@ -98,7 +98,26 @@ static void Main(string[] args)
                     {
                         Console.WriteLine("(response not recognized)");
                     }
-                } while (intersectionChoice.ToLower() != "left"); 
+                } while (intersectionChoice.ToLower() != "left");
+
+                string response2 = GetUserResponse("Do you want to follow him?");
```

Comment:
When variables start having numbers after them, it's an indication that their names need to be re-evaluated.  Consider either re-using the `response` variable, or try naming them more specifically for their purpose; perhaps "followResponse" or something along those lines.  Same for `response3` on line 109